src/bot/scripts/card.js

```diff
@@ -728,946 +66,1233 @@ const card = new Command({
+                    cardData.aoe_damage_percentage * 100
+                  ) / 100}\n`;
+                  embed.description += `AOE Knockback Percentage: ${Math.round(
+                    cardData.aoe_knockback_percentage * 100
+                  ) / 100}\n`;
+                  embed.description += `AOE Radius: ${Math.round(
+                    cardData.aoe_radius * 100
+                  ) / 100}\n\n`;
+                } else {
+                  // no aoe
+
+                  embed.description += "**AOE Attacks? - No**\n\n";
+                }
+
+                embed.description += `Full Stats: https://sppd.feinwaru.com/${cardData.image}`;
+
+                embed.setFooter("© 2018 Copyright: Feinwaru Software ");
+
+                // ***ATTACK INFO***
+
+                // -can attack
+                // -attack range
+                // -time in between attacks
+                // -pre attack delay
+                // -knockback impulse
+                // -aoe attack type
+                // -aoe damage percentage
+                // -aoe radius
+                // -aoe knockback percentage
+
+                // ***POWER INFO***
+
+                // -targeting ...
+                // -power duration
+
+                // ***SPEED INFO***
+
+                // -time to max velocity
+                // -max velocity
+
+                // ***totem ONLY***
+
+                // -health loss
+
+                // ***spell ONLY***
+
+                // *** ??? ***
+
+                // -requirements ...
+                // -child unit limit
+
+                // *name
+                // CanAttack
+                // *description
+                // *Image
+                // -ManaCost
+                // -damage
+                // -health
+                // healthLoss - character_type: totem
+                // type - if !Character, character_type === undefined
+                // Targeting ... - power radius
+                // character_type
+                // AttackRange
+                // TimeToMaxVelocity
+                // MaxVelocity
+                // TimeInBetweenAttacks
+                // PowerDuration
+                // -PowerPower
+                // -rarity
+                // -theme
+                // Requirements
+                // AOEAttacktype
+                // AOEdamagePercentage
+                // AOERadius
+                // AOEKnockbackPercentage
+                // PreAttackDelay
+                // CastArea - ownside/anywhere
+                // ChildUnitLimit
+
+                await message.channel.send("", {
+                  file: path.join(__dirname, "temp", `pd-${saveDate}.png`)
+                });
+                message.channel.send(embed);
+                fs.unlink(
+                  path.join(__dirname, "temp", `pd-${saveDate}.png`),
+                  error => {
+                    if (error !== null && error !== undefined) {
```

Comment:
ik this might have been here before, but when comparing against null/undefined, do x != null or x == null

Reply:
Done